tests/test_class_generate_help.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import, division, print_function
+
+import pytest
+
+import environ
+
+
+@environ.config(prefix="APP")
+class AppConfig(object):
+    host = environ.var("127.0.0.1", help="host help")
+    port = environ.var(5000, converter=int, help="port help")
+
+
+@environ.config(prefix="APP", generate_help="gen_help")
+class ConfigRenamed(object):
+    host = environ.var("127.0.0.1", help="host help")
+    port = environ.var(5000, converter=int, help="port help")
+
+
+@environ.config(prefix="APP", generate_help="")
+class ConfigEmptyName(object):
+    host = environ.var("127.0.0.1", help="host help")
+    port = environ.var(5000, converter=int, help="port help")
+
+
+@environ.config(prefix="APP", generate_help=None)
+class ConfigNoneName(object):
+    host = environ.var("127.0.0.1", help="host help")
+    port = environ.var(5000, converter=int, help="port help")
+
+
+def test_has_classmethod():
+    """
+    Class based `generate_help` classmethod exists
+    """
+    assert hasattr(AppConfig, "generate_help")
```

Comment:
Pease don't use `hasattr()` because [it can mask problems](https://hynek.me/articles/hasattr/).

Ideally create a sentinel at the beginning like `sentinel = object()` and then make is either `assert getattr(AppConfig, X, sentinel) is sentinel` or `... is not sentinel`.

Reply:
Done. Again - thanks on hasattr lesson.